sklearn/metrics/tests/test_regression.py

```diff
@@ -577,7 +586,7 @@ def objective_func(x):
 def test_dummy_quantile_parameter_tuning():
     # Integration test to check that it is possible to use the pinball loss to
     # tune the hyperparameter of a quantile regressor. This is conceptually
-    # similar to the previous test but using the scikit-learn estimator and
+    # similar to the previous test but using the scikit-learn metric and
```

Comment:
I think "estimator" is the right word here. The metric is being used via the `DummyRegressor` and not directly.